.github/workflows/dotnetcoretest.yml

```diff
@@ -39,9 +39,9 @@ jobs:
         run: dotnet build --configuration Release --no-restore
       - name: Test (Linux)
         working-directory: ${{ runner.workspace }}/Lightning.NET
-        run: LD_LIBRARY_PATH=${{ runner.workspace}}/Lightning.NET/lmdb/libraries/liblmdb/:$LD_LIBRARY_PATH dotnet test -f net6.0 --no-restore --verbosity normal
+        run: LD_LIBRARY_PATH=${{ runner.workspace}}/Lightning.NET/lmdb/libraries/liblmdb/:$LD_LIBRARY_PATH dotnet test -f net7.0 --no-restore --verbosity normal
         if: matrix.os == 'ubuntu-latest'
       - name: Test (Windows or Mac)
         working-directory: ${{ runner.workspace }}/Lightning.NET
-        run: dotnet test -f net6.0 --no-restore --verbosity normal
+        run: dotnet test -f net7.0 --no-restore --verbosity normal
```

Comment:
I'd test also on net6.0 as it's a supported version

Reply:
I'm assuming the setup-dotnet dotnet-version when multiple versions are provided I can lean on ignoring the -f option altogether and it will run for both versions I've specified?

Reply:
Looks like the build confirms my thoughts.